albumentations/augmentations/transforms.py

```diff
@@ -318,8 +318,12 @@ class ImageCompression(ImageOnlyTransform):
     """Decreases image quality by Jpeg, WebP compression of an image.
 
     Args:
-        quality_lower: lower bound on the image quality. Should be in [0, 100] range for jpeg and [1, 100] for webp.
-        quality_upper: upper bound on the image quality. Should be in [0, 100] range for jpeg and [1, 100] for webp.
+        quality_lower: **Deprecated: use 'quality_range' instead** lower bound on the image quality.
+            Should be in [0, 100] range for jpeg and [1, 100] for webp.
+        quality_upper: **Deprecated: use 'quality_range' instead** upper bound on the image quality.
+            Should be in [0, 100] range for jpeg and [1, 100] for webp.
+        quality_range: tuple of bounds on the image quality i.e. (quality_lower, quality_upper).
+            Both values should be in [0, 100] range for jpeg and [1, 100] for webp.
```

Comment:
Just remove `quality_lower` and `quality_upper` from docstring. 

If someone uses them -> that's fine, but it could be better if all new users would not even know about such option.